HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -7932,6 +7981,46 @@ Patch "Keep Red Rings on Death" in "Cheats" by "WasifBoomz" does "Prevents you f
     );
 }
 
+Code "Lose Speed on Collision" in "Physics" by "WasifBoomz" does "Reduce speed when you hit a wall, rather than keeping your speed."
+//
+    #lib "BlackboardStatus"
+    #lib "Sonic"
+
+    #load "System.Numerics.dll"
+	
+    using System;
+    using System.Numerics;
+
+    static float* deltaTime = (float*)0x14F26C;
```

Comment:
Use the `Time` library for delta time using `Time.GetDeltaTime()`.

Never use raw addresses like this.